setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+from setuptools import setup
+from distutils.util import convert_path
+import os
+
+main_ns = {}
+ver_path = convert_path('src/version.py')
+with open(ver_path) as ver_file:
+    exec(ver_file.read(), main_ns)
+VER = main_ns['__version__']
+
+requirementPath = os.path.dirname(os.path.realpath(__file__)) + '/requirements.txt'
+install_requires = []
+if os.path.isfile(requirementPath):
+    with open(requirementPath) as f:
+        install_requires = f.read().splitlines()
+
+setup(
+    name='soundbay',
+    version=VER,
+    packages=setuptools.find_packages(),
+    install_requires=install_requires,
```

Comment:
perhaps we should add version to every package we use? Might be safe, so we won't end up with errors in dependency

Reply:
Would try to add <> condtions, we don't want setup to be restrictive